tests/test_perseptual.py

```diff
@@ -0,0 +1,55 @@
+import torch
+import pytest
+from typing import Any, List
+
+from photosynthesis_metrics import ContentLoss, StyleLoss, LPIPS
+
+
+@pytest.fixture(scope='module')
+def prediction() -> torch.Tensor:
+    return torch.rand(4, 3, 128, 128)
+
+
+@pytest.fixture(scope='module')
+def target() -> torch.Tensor:
+    return torch.rand(4, 3, 128, 128)
+
+
+devices: List[str] = ["cpu"]
+if torch.cuda.is_available():
+    devices.append("cuda")
+
+
+@pytest.fixture(scope='module', params=devices)
+def device(request: Any) -> Any:
+    return request.param
+
+
+# ================== Test class: `ContentLoss` ==================
+def test_content_loss_init(prediction: torch.Tensor, target: torch.Tensor) -> None:
```

Comment:
Do we need such test as the initialisation is checked in forward test?
If so, the arguments are not needed.
Same for the following losses.